Switch to `conntrack-tools` as kubelet dependency for RPM's

saschagrunert opened the pull request:
#### What type of PR is this?

/kind bug


#### What this PR does / why we need it:
The alias `conntrack-tools` should be available for most RPM based distributions, including the mentioned (open)SUSE ones.

#### Which issue(s) this PR fixes:

<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.

Fixes #

or

None
-->
Fixes https://github.com/kubernetes/release/issues/3711
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

<!--
If no, just write "NONE" in the release-note block below.
If yes, a release note is required:
Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".

For more information on release notes see: https://git.k8s.io/community/contributors/guide/release-notes.md
-->

```release-note
Switch to `conntrack-tools` as kubelet package dependency for RPM's.
```

Muthukumar-Subramaniam commented:
Hi @saschagrunert , I have updated a consideration regarding debian based distros in https://github.com/kubernetes/release/issues/3711
Please do consider it before this pull request gets merged.

saschagrunert commented:
/hold

saschagrunert commented:
> Hi @saschagrunert , I have updated a consideration regarding debian based distros in #3711 Please do consider it before this pull request gets merged.

Thank you, I've updated the PR accordingly.

/unhold

xmudrii reviewed (approved):
Thank you!
/lgtm
/approve
